### PR TITLE
Create team-culture.md

### PR DESCRIPTION
**Pull request by amenne**
Re-organization of CE space; added sub pages to capture procedures and practices, processes, and CE competencies

### CODE REVIEW

File: handbook/ce/team-culture.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+We believe that this is the exact moment in our journey to take such a risk and we bet that if we carve out focus and space, we will achieve the following benefits: matured capabilities (eg, demos, trials, data-driven customer health measurement and analysis, documentation, etc), increased footprint (eg, strategic account growth, dev community advocacy and outreach, building customer-centric solutions to expand Sourcegraph, etc), and internal opportunities for CE growth and advancement (eg, strategic focus, PM-type work, Eng-type build work, etc). 
+
+### Future Vision
+We envision our CE team being able to continue building and fostering relationships with developer users and champions at our prospects and customers, continue serving as the technical experts during the sales cycle and helping enable our customers throughout our journey together (from the very first meeting and every subsequent meeting thereafter), while being able to impact and deliver cross-functional, cross-customer projects and establish a new strategic customer focus.
+
+##### What we mean by cross-functional, cross-customer projects
+A project-based focused falls into two categories:
```
Comment (emchap) [inline handbook/ce/team-culture.md:118]: Is this supposed to be "Project-based work" possibly? Or "The focus areas for a project-based CE"? "A project-based" feels like it was intended to have a noun after it.

File: handbook/ce/team-culture.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+As we continue CE hiring and onboarding in Q2 and beyond, and as we assess both the needs and impact of this project work for our teams and our customers, we will gauge whether there exists enough need to have CEs be exclusively dedicated to project work in the future.
+
+For CEs to be able to execute on these critically important areas will require availability and focus. As such, this exercise has brought to light the need to clarify and/or realign some specific areas to other parts of the company.
+
+#### Teams with whom we have aligned
+Product:
+* For NPS feedback, PMs should review and follow up. For our strategic accounts, Product will work with the AE and the CE to address any feedback and participate in weekly product roadmap reviews or monthly feedback theme reviews.
```
Comment (emchap) [inline handbook/ce/team-culture.md:144]: Does "follow-up" here entail also emailing the customer? (Right now I do some amount of "hi, customer, you said we can't do this and we actually can, here's the doc.")

File: handbook/ce/working-with-customers.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Customer Engineering :: Working with Customers
```
Comment (emchap) [inline handbook/ce/working-with-customers.md:1]:
```suggestion
# Customer Engineering: Working with Customers
```

File: handbook/ce/working-with-customers.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+#### Red Accounts Process
+In slack, the #red-accounts channel has been created so that when the Customer Health field on the Salesforce Account record is set to Red, a post is auto-triggered in the slack channel and a corresponding row will be automatically added to the Account Tracker tab of the [Red Accounts](https://docs.google.com/spreadsheets/d/1eVgWhrtgH8WQGo_pRuMseqz-Bk1P1Bymrlkutzz5jEA/edit#gid=0) Google Sheet. The AE / CE will jointly fill in relevant information to understand current state, needs, and the intended actions. Any asks or needs against the intended action plan should be initiated via a thread on the auto-generated slack post; this allows for visibility and transparency.
+
+#### Roles and Responsibilities
+The CE will monitor the Customer Health dashboard and where appliable update the Customer Health field on the Salesforce Account record to red.
+
+Should an AE or CE feel that an account, which isn't designated as red via the health dashboard, is in fact red for any reason (eg a champion leaves, etc) they should align, and the CE should update the Salesforce Account record to red.
```
Comment (emchap) [inline handbook/ce/working-with-customers.md:47]:
```suggestion
Should an AE or CE feel that an account which isn't designated as red via the health dashboard, is in fact red for any reason (eg a champion leaves, etc) they should align, and the CE should update the Salesforce Account record to red.
```

File: handbook/ce/team-culture.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+* CE team meeting (bi-weekly on Wednesdays): Global team sync; open team discussion about any topics
+* Demo workshop (bi-weekly on Wednesdays): A space for the CE team to collaborate on demos, scripts, and other ideas / needs that will help us serve our customers better
+* CE team hangout (monthly on Thursdays): An open forum to ask questions, collaborate, and chat as a team.
+* Product & CE Requests & Feedback Themes (bi-weekly on Wednesdays): Feedback treads and requests between CE and Product.
+* CE <> Sales Office Hours (bi-weekly on Tuesdays): An open forum to discuss topics and answer questions from the sales team.
+
+
+
+## OOO Protocols
```
Comment (emchap) [inline handbook/ce/team-culture.md:66]:
```suggestion
## OOO/Vacation/PTO Protocols
```

Comment (emchap) [reply handbook/ce/team-culture.md:66]: Wide smattering of key terms here for discoverability.

**Review by emchap — COMMENTED** [review]
Looks great! Left a few suggestions.

File: handbook/ce/team-culture.md (hunk 2 above)
Comment (amenne) [reply handbook/ce/team-culture.md:144]: The intent is for product to reach out to customers. Of course can always work with AE / CE to determine best strategy.

File: handbook/ce/team-culture.md (hunk 1 above)
Comment (amenne) [reply handbook/ce/team-culture.md:118]: I don't see this. What line? If 118, how about just: "Project-based focused falls into two categories:"

File: handbook/ce/team-culture.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+* [Slack Channels](#ce-specific-slack-channels)
+* [CE Forward-Looking Vision](#forward-vision)
+
+--------
+
+## How we intersect other teams
+We work alongside just about every team at Sourcegraph. Here is how we add value to each other and a collaboration overview for each pairing.
+
+### Sales
+- **How CE adds value to sales:** CEs are the right-hand team for Sales by serving as technical experts; CEs lead product demos, guide customers through trials/POCs, lead production deployments, focuses on product usage and adoption.
```
Comment (christyharagan) [inline handbook/ce/team-culture.md:18]: Add: builds technical champions

File: handbook/ce/team-culture.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+    - Customer Name:
+    - Link to issue (slack / GH / ZD ticket #):
+    - Background information or anything we should know that would be helpful?:
+  - CE may also need to share additional context around an issue with the head of support. For example, a CE may need to share special circumstances around a customer issue. In scenarios such as this, the CE should raise a request in the support channel (via the Support Ticket Request shortcut).  For each request the CE should provide the following information:
```
Comment (christyharagan) [inline handbook/ce/team-culture.md:30]: Do we document somewhere the Slack commands for raising this context?

File: handbook/ce/team-culture.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+    - Customer Name:
+    - Link to issue (Slack / Github issue / ZD ticket #):
+    - What’s the ask:
+    - Why are we making this ask:
+    - What happens if we don’t take this desired action:
+    - Is there a time consideration we need to be aware of, if so what is it and why?:
+    - Anything else we should know that would be helpful?:
+
+### Software Engineers (SWEs)
+- **How CE adds value to SWEs:** CE provides important insights from prospective and current customers which inform and serve as important inputs to SWE roadmap.
```
Comment (christyharagan) [inline handbook/ce/team-culture.md:40]: Isn't roadmap owned by Product?

File: handbook/ce/team-culture.md [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+- **How SWEs adds value to CE:** SWEs create a high quality product and when needed, helps educate CEs on how certain features work so that they can educate customers. SWEs also conduct via planned training sessions, periodic pairing, deep-dives on new features/products, etc.
+- **Collaboration overview:** CEs can pose how-to questions and provide feedback via Slack
+
+### Product
+- **How CE adds value to product:** CEs provide a data-driven view into customer concerns/questions, ad-hoc feedback shared whilst helping customers, and helps update documentation so customers (and we) have a single source of truth.
+- **How product adds value to CE:** Product educates CE on new features and helps clarify expected behavior questions.
+- **Collaboration overview:** CEs provide Product with feedback and feature requests; Product seeks inputs from customer through CEs.
+
+### Marketing
+- **How CE adds value to marketing:** CEs help ensure that when developers think of Sourcegraph, they associate it with quality and responsiveness.
```
Comment (christyharagan) [inline handbook/ce/team-culture.md:50]: CEs help ensure marketing material is relevant and valuable to engineers and engineering management

File: handbook/ce/team-culture.md (hunk 7 above)
Comment (amenne) [reply handbook/ce/team-culture.md:30]: Yes above: (via the Support Ticket Context shortcut) and (via the Support Ticket Request shortcut).  Those are the names of the shortcuts

File: handbook/ce/team-culture.md (hunk 6 above)
Comment (amenne) [reply handbook/ce/team-culture.md:18]: Where do you suggest this be added? Under how we add value to sales?

File: handbook/ce/team-culture.md (hunk 1 above)
Comment (emchap) [reply handbook/ce/team-culture.md:118]: Sorry, yes, line 118: "A project-based focused falls into two categories" just seemed like it was missing the noun.

Comment (emchap) [reply handbook/ce/team-culture.md:118]:
```suggestion
Project-based work falls into two categories:
```

File: handbook/ce/team-culture.md (hunk 2 above)
Comment (emchap) [reply handbook/ce/team-culture.md:144]: Fair enough! From my perspective: Product won't want to respond to everything (sometimes it's just a confused individual user) and that's fine, but I want to make sure the individual users in my accounts get the info they need from a success perspective, so I'd prefer to respond to them. But perhaps I'm missing something/am offbase here.

File: handbook/ce/team-culture.md (hunk 1 above)
Comment (amenne) [reply handbook/ce/team-culture.md:118]: Ah thank you for this change - you rock!!!

**Pull request merged**